packages/tinacms/src/toolkit/fields/plugins/mdx-field-plugin/plate/components/plate-ui/mermaid-element.tsx

```diff
@@ -0,0 +1,94 @@
+import React, { useEffect, useState } from 'react'
+import { withRef } from '@udecode/cn'
+import { PlateElement } from '@udecode/plate-common'
+import { useMermaidElement } from '../../hooks/use-mermaid-element'
+import { Moon, PencilIcon, SunMoon } from 'lucide-react'
+import { CodeBlock } from '../../plugins/ui/code-block'
+import { ELEMENT_MERMAID } from '../../plugins/custom/mermaid-plugin'
+
+const LightModeComponent = ({ onToggleMode }) => {
+  const [isLightMode, setIsLightMode] = useState(false)
+
+  const handleToggle = (e) => {
+    e.preventDefault()
+    setIsLightMode((prevMode) => !prevMode)
+  }
+
+  useEffect(() => {
+    //? Note: Adding this class on a <pre/> will remove its base bg styling
```

Comment:
**nitpick**: Stray '?'
```suggestion
    // Note: Adding this class on a <pre/> will remove its base bg styling
```

Reply:
Ah this is due to an extension that I use - https://marketplace.visualstudio.com/items?itemName=aaron-bond.better-comments 
Makes the comment blue when adding a ?